spec/GraphHopperOptimizationSpec.js

```diff
@@ -1,64 +1,31 @@
-describe("js-client", function () {
+var GraphHopperOptimization = require('../src/GraphHopperOptimization');
+var ghOptimization = new GraphHopperOptimization({key: key, vehicle: profile});
 
-    var ghOptimization;
-    var fixtures;
-
-    beforeEach(function () {
-        // load multiple json fixtures via comma separated
-        fixtures = loadJSONFixtures('response1.json');
-
-        jasmine.Ajax.install();
-        ghOptimization = new GraphHopper.Optimization({key: "defaultKey", profile: "car", waitInMillis: 0});
-    });
-
-    it("should parse services correctly", function () {
-
-        var response = {};
-        var callback = function (json) {
-            response.json = json;
-        };
+describe("Optimization Test", function () {
+    it("Get results", function (done) {
 
         var jsonInput = {
             "vehicles": [{
-                    "vehicle_id": "vehicle1",
-                    "start_address": {"location_id": "v1_startLoc", "lon": 9.797058, "lat": 52.375599}
-                }],
+                "vehicle_id": "vehicle1",
+                "start_address": {"location_id": "v1_startLoc", "lon": 9.797058, "lat": 52.375599}
+            }],
             "services": [{
-                    "id": "b1",
-                    "name": "drink_bionade_in_rostock",
-                    "address": {"location_id": "loc_b1", "lon": 12.1333333, "lat": 54.0833333}
-                }, {
-                    "id": "b2",
-                    "name": "drink_cola_in_berlin",
-                    "address": {"location_id": "loc_b2", "lon": 13.354568, "lat": 52.514549}
-                }]
+                "id": "b1",
+                "name": "drink_bionade_in_rostock",
+                "address": {"location_id": "loc_b1", "lon": 12.1333333, "lat": 54.0833333}
+            }, {
+                "id": "b2",
+                "name": "drink_cola_in_berlin",
+                "address": {"location_id": "loc_b2", "lon": 13.354568, "lat": 52.514549}
+            }]
         };
 
-        ghOptimization.doRequest(jsonInput, callback, {});
-        request = jasmine.Ajax.requests.mostRecent();
-
-        expect(request.data()).toEqual(jsonInput);
-        expect(request.url).toBe('https://graphhopper.com/api/1/vrp/optimize?key=defaultKey');
-        expect(request.method).toBe('POST');
-
-        request.respondWith({status: 200, responseText: '{"job_id":"018a5181-d893-4373-a7de-ebdc38617fcb"}'});
-
-        request = jasmine.Ajax.requests.mostRecent();
-        expect(request.method).toBe('GET');
-        expect(request.url).toBe('https://graphhopper.com/api/1/vrp/solution/018a5181-d893-4373-a7de-ebdc38617fcb?key=defaultKey');
+        ghOptimization.doRequest(jsonInput, function (json) {
+            expect(json.message).not.toBeDefined();
+            expect(json.solution).not.toBeLessThan(0);
```

Comment:
For the solution we should assert that it is one vehicle plus two services? and the time estimate